Xamarin.Forms.Material.iOS/MaterialActivityIndicatorRenderer.cs

```diff
@@ -80,11 +81,24 @@ public override void LayoutSubviews()
 			if (_center != Control.Center)
 			{
 				_center = Control.Center;
-				_backgroundLayer.Path = UIBezierPath.FromArc(_center, Control.Radius - 2, 0, 360, true).CGPath;
+				_backgroundLayer.LineWidth = Control.StrokeWidth;
+				var pathRadius = Control.Radius - Control.StrokeWidth / 2;
+				_backgroundLayer.Path = UIBezierPath.FromArc(_center, pathRadius, 0, 360, true).CGPath;
 			}
 			SetBackgroundColor(Element.BackgroundColor);
 		}
 
+		public override CGSize SizeThatFits(CGSize size)
+		{
+			var radius = NMath.Min(size.Width, size.Height) / 2;
+			if (!nfloat.IsInfinity(radius))
+			{
+				Control.Radius = radius;
+				Control.StrokeWidth = Control.Radius / 6;
```

Comment:
Maybe move this 6 into a constant so that it has a human name for future reference.

Reply:
I'm not sure that this is needed, this is a computed constant only needed here.
By spec `StrokeWidth` is equal 4 with diameter is 48... For scaling `StrokeWidth` = diameter / 12 = radius / 6. As in Android.